src/components/MetadataSources/MetadataSources.test.js

```diff
@@ -20,73 +21,113 @@ const filterData = { contacts: [
   }
 ] };
 
-const testSource = {
-  logger: { log: noop },
-  mutator: { sources: {}, query: {}, resultCount: {} },
-  props: { history: {}, location: {}, match: {}, staticContext: undefined, children: {} },
-  recordsObj: {},
-  resources: {
-    sources: {},
-    query: { query: '', filters: 'status.active,status.implementation', sort: 'label' },
-    resultCount: 30
-  }
-};
-
-const connectedTestSource = new StripesConnectedSource(testSource.props, testSource.logger, 'sources');
-
-const renderMetadataSources = (stripes) => (
-  render(withIntlConfiguration(
-    <Router>
-      <StripesContext.Provider value={stripes}>
-        <MetadataSources
-          contentData={metadatasources}
-          source={connectedTestSource}
-          filterData={filterData}
-          onNeedMoreData={jest.fn()}
-          queryGetter={jest.fn()}
-          querySetter={jest.fn()}
-          searchString="status.active,status.implementation"
-          selectedRecordId=""
-          onChangeIndex={jest.fn()}
-        />
-      </StripesContext.Provider>
-    </Router>
-  ))
+const renderMetadataSources = (stripes, data) => withIntlConfiguration(
+  <MemoryRouter>
+    <StripesContext.Provider value={stripes}>
+      <MetadataSources
+        contentData={data}
+        filterData={filterData}
+        onNeedMoreData={jest.fn()}
+        queryGetter={jest.fn()}
+        querySetter={jest.fn()}
+        searchString="status.active,status.implementation"
+        selectedRecordId=""
+        visibleColumns={['label', 'sourceId', 'status', 'lastProcessed']}
```

Comment:
`<MetadataSources>` does not define a `visibleColumns` prop

Reply:
Removed `visibleColumns`